README.md

```diff
@@ -80,84 +243,18 @@ The tool itself is launched via the following line:
 
 The `<dataset_name>.csv`, which is a user-provided dataset, should be placed in the `/path/to/Desbordante/build/target` directory.
```

Comment:
Should be removed

Reply:
Also:
The tool itself is launched via the following line: -> The tool itself may be run like the following:
<dataset_name> -> <path_to_dataset>